playeur.py

```diff
@@ -0,0 +1,9 @@
+
+class Playeur:
```

Comment:
Y a une petite faute d'orthographe dans le nom de la classe ! A voir si tu peux renommer le fichier aussi. 

```suggestion
class Player:
```